tests/pypiserver/start.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+set -o errexit -o nounset -o pipefail
+
+DIR=$(dirname "$(realpath "${BASH_SOURCE[0]}")")
+
+DEFAULT_IMAGE=docker.io/pypiserver/pypiserver:v2.1.1@sha256:17198f668ef4f460ee81456f09fc65d352766fa9e49a81474304c5aa69b8be38
+DEFAULT_PORT=8080
+
+cleanup() {
+    rm -r "$WORKDIR" || true
+    podman volume rm -f --time 0 cachi2-pypiserver-packages >/dev/null || true
+    podman volume rm -f --time 0 cachi2-pypiserver-auth >/dev/null || true
+}
+
+WORKDIR=$(mktemp -d --tmpdir "cachi2-pypiserver.XXXXXX")
+trap cleanup EXIT
+
+setup() {
+    echo -e "\n--- Downloading $DIR/package-urls.txt ---\n"
+
+    mkdir "$WORKDIR/packages"
+    sed '/^#/d' "$DIR/package-urls.txt" |
```

Comment:
wouldn't `tail -n +2` to skip the first line be actually more straightforward and slightly more readable?

Reply:
I went with a "comment" syntax, in case I wanted to add more packages to download (and more comments explaning how I got the urls)